Quaver.Shared/Screens/Selection/SelectionScreen.cs

```diff
@@ -376,7 +376,7 @@ private void HandleKeyPressEnter()
                         ActiveScrollContainer.Value = SelectScrollContainerType.Maps;
                     break;
                 case SelectScrollContainerType.Maps:
-                    ExitToGameplay();
+                        ExitToGameplay();
```

Comment:
Please revert this indentation
```suggestion
                    ExitToGameplay();
```